Gemfile.lock

```diff
@@ -266,23 +265,21 @@ GEM
     mini_portile2 (2.8.0)
     minitest (5.16.3)
     mqtt (0.5.0)
-    msgpack (1.5.6)
+    msgpack (1.6.0)
     multi_json (1.15.0)
-    mustermann (2.0.2)
+    mustermann (3.0.0)
```

Comment:
No code changes; Just dropping support for ruby 2.2 https://my.diffend.io/gems/mustermann/2.0.2/3.0.0#d2h-357803